sickrage/providers/GenericProvider.py

```diff
@@ -264,7 +266,7 @@ def find_search_results(self, show, episodes, search_mode, forced_search=False,
                 logger.log(u'Adding item from search to cache: %s' % title, logger.DEBUG)
                 # pylint: disable=protected-access
                 # Access to a protected member of a client class
-                ci = self.cache._addCacheEntry(title, url, seeders, leechers, size, pubdate, hash, parse_result=parse_result)
+                ci = self.cache._addCacheEntry(title, url, seeders, leechers, size, pubdate, torrent_hash)
```

Comment:
This is the actual bug. addCacheEntry doesn't provide paramater parse_result anymore.